Website/src/rendering/src/assets/css/layout/grids.css

```diff
@@ -29,6 +29,15 @@
     .grid-item {
       @apply rounded overflow-hidden;
 
+      transform: scale(1);
+      transition: all 0.1s;
+
+      &:hover {
+        box-shadow: 1px 1px 10px rgba(23, 23, 23, 0.45);
```

Comment:
This change affects ALL the grids everywhere on the website.
Have you checked the implications in Chromatic?

For example, here are some places where it looks odd:
Shop page:
![image](https://user-images.githubusercontent.com/2000200/134946804-0a662c6f-b2b2-4fb4-8162-5689d0ebcc23.png)

Home page > Speakers:
![image](https://user-images.githubusercontent.com/2000200/134946889-ee8ce777-420a-4d4b-82dd-b8a7ab98a192.png)

But speakers are different on the Speakes page:
![image](https://user-images.githubusercontent.com/2000200/134946995-6137c55f-73cc-46be-9818-df622330e481.png)

News do not have any hover effect:
![image](https://user-images.githubusercontent.com/2000200/134947158-6f9b314b-0e2f-4774-b3b2-d0677184b237.png)